scispacy/custom_sentence_segmenter.py

```diff
@@ -31,32 +15,23 @@ def combined_rule_sentence_segmenter(doc: Doc) -> Doc:
 
     @param doc: the spaCy document to be annotated with sentence boundaries
     """
-    segmenter = pysbd.Segmenter(language='en', clean=False)
-    segments = segmenter.segment(doc.text)
-    segments = merge_segments(segments)
-
-    # pysbd splits raw text into sentences, so we have to do our best to align those
-    # segments with spacy tokens
-    segment_index = 0
-    current_segment = segments[segment_index]
-    built_up_sentence = ""
-    for i, token in enumerate(doc):
-        if i == 0 and (token.is_space or token.text == '.'):
```

Comment:
Could you add a test for what happens when the document starts with whitespace?

Reply:
starting whitespace(s) would be separate sentences.

```python
combined_rule_sentence_segmenter("    This document starts with whitespaces. Next sentence.") 
["    ", "This document starts with whitespaces.", "Next sentence."]
```

This allows us to recreate original text

Reply:
:+1: